mappings/net/minecraft/world/Difficulty.mapping

```diff
@@ -4,10 +4,11 @@ CLASS ahb net/minecraft/world/Difficulty
 	FIELD g translationKey Ljava/lang/String;
 	METHOD <init> (Ljava/lang/String;IILjava/lang/String;)V
 		ARG 3 id
+		ARG 4 translationKey
 	METHOD a getId ()I
 	METHOD a getDifficulty (I)Lahb;
 		ARG 0 difficulty
 	METHOD a getDifficulty (Ljava/lang/String;)Lahb;
 		ARG 0 translationKey
-	METHOD b toTextComponent ()Ljm;
+	METHOD b getName ()Ljm;
```

Comment:
Consider unifying `getTranslatbleName`, `getName`, and `toHoverableText`/`toText`